Aptoide/appcoins-wallet-android
# [WAL-70] Remove gamification level up highlight

**What does this PR do?**

Needs WAL-54 to be accepted
Removes almost every reference and logic related to gamification level up highlight
Keeps the close to level up logic in the Gamification Interactor to later be used.

**Database changed?**

   No

**Where should the reviewer start?**

PaymentMethodsPresenter.kt

**How should this be manually tested?**

With a close to level up wallet 249 (appc spent), check if the normal bonus layout is shown.

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/WAL-70

**Code Review Checklist**
